src/api/public/apidocs-new/paths/attribute_namespace_attribute_name_meta.yaml

```diff
@@ -0,0 +1,199 @@
+          examples:
+            namespace:
+              value:
+                code: not_found
+                summary: Couldn't find AttribNamespace
+              summary: Not Found (Namespace)
+            unknown_attribute:
+              value:
+                code: unknown_attribute
+                summary: "Unknown attribute 'OBS_TEST':'OwnerRootProjectTest'"
+              summary: Unknown Attribute
+  tags:
+    - Attributes
+
+post:
+  # 'post' and 'put' operations must be in sync
+  summary: Change attribute data. Create an attribute if it doesn't exist.
+  description: |
+    Change attribute data. Create an attribute if it doesn't exist.
+
+    This operation is the same as the one defined with [PUT](#/Attributes/put_attribute__namespace___attribute_name___meta).
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/namespace.yaml'
+    - $ref: '../components/parameters/attribute_name.yaml'
+  requestBody:
+    description: Attribute definition.
+    required: true
+    content:
+      application/xml; charset=utf-8:
+        schema:
+          $ref: '../components/schemas/attribute.yaml'
+  responses:
+    '200':
+      $ref: '../components/responses/succeeded.yaml'
+    '400':
+      description: Validation Failed.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            empty_body:
+              value:
+                code: validation_failed
+                summary: Document is empty, not allowed for attrib_type
+              summary: Validation Failed (Empty Body)
+            wrong_xml_element:
+              value:
+                code: validation_failed
+                summary: 'attrib_type validation error: 1:0: ERROR: Element definition failed to validate attributes'
+              summary: Validation Failed (Wrong XML Attributes)
+            illegal_request:
+              value:
+                code: illegal_request
+                summary: 'Illegal request: PUT/POST /attribute/OBS_TEST/OwnerRootProjectTest/_meta: path does not match content'
+              summary: Illegal Request
+    '401':
+      $ref: '../components/responses/unauthorized.yaml'
+    '404':
+      description: Not Found.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            namespace:
+              value:
+                code: not_found
+                summary: Couldn't find AttribNamespace
+              summary: Not Found (Namespace)
+            unknown_attribute:
+              value:
+                code: unknown_attribute
+                summary: "Unknown attribute 'OBS_TEST':'OwnerRootProjectTest'"
+              summary: Unknown Attribute
+  tags:
+    - Attributes
+
+put:
+  # 'post' and 'put' operations must be in sync
```

Comment:
Why document both? What's the benefit?

Reply:
In the case of documenting only one action, people will think the other action is not defined.

I prefer to have duplicate API endpoint documentation, easy to search for an action on a resource (the Swagger-UI way), than to provide documentation that forces the user to search for all the actions in a resource, to find the specific action he is interested in.

And if we only document one action, in this particular `post/put` case, what one should be chosen to be documented?

Reply:
You can document post and put something like:

> Note: This route is also accessible via the HTTP PUT method

Reply:
We discussed about this for a while but couldn't come to an agreement. This needs further discussion together with everybody, but this shouldn't block this PR.